movement/watch_faces/complications/countdown_face.h

```diff
@@ -0,0 +1,43 @@
+#ifndef COUNTDOWN_FACE_H_
+#define COUNTDOWN_FACE_H_
+
+#include "movement.h"
+
+/*
+A countdown/timer face
+
+Max countdown is 99 minutes and 59 seconds since we have to prevent the watch
+from going to deep sleep using movement_schedule_background_task
+*/
+
+
+typedef enum {
+    cd_waiting,
+    cd_running,
+    cd_setting
+} countdown_mode_t;
```

Comment:
Is there a way to have a static enum so that these don't contaminate the global namespace? Or maybe I'm just being too paranoid

Reply:
I think as long as they're reasonably prefixed, it's nothing to worry about. My instinct would be to do something like `countdown_mode_waiting` here, but the `cd_` prefix seems fine!